evadb/configuration/constants.py

```diff
@@ -28,5 +28,6 @@
 CACHE_DIR = "cache"
 DATASET_DATAFRAME_NAME = "dataset"
 DB_DEFAULT_NAME = "evadb.db"
+PG_DB_DEFAULT_NAME = "circle_test"
```

Comment:
If this is only for `circleci` testing, we can hardcode it.

Reply:
There is some issue in circleci wherein the postgres image pulled is not honoring the environment variable for db_name, and is instead getting defaulted to 'circle_test'. Hence I named it as such here. Should I still change it, for local workflows, the constant can be changed as desired.

Reply:
There is already a bug open on this : https://github.com/CircleCI-Public/cimg-postgres/issues/94